src/services/followMainFeedBroad.js

```diff
@@ -1,41 +1,39 @@
-const stream = require("getstream");
-const { unFollowFeedProcessQueue } = require("../config");
+const stream = require('getstream');
+const {unFollowFeedProcessQueue} = require('../config');
 
-const followMainFeedBroad = async (userId, userIds) => {
-  console.log("Broad Feed => ", userIds)
-  if (!userIds || userIds.length == 0) {
+const followMainFeedBroad = async (userId, userIds, user_anon = false) => {
+  if (!userIds || userIds.length === 0) {
     return;
   }
-
+  const targetFeed = user_anon ? 'user_anon' : 'user';
   const cs = stream.connect(process.env.API_KEY, process.env.SECRET);
 
   const payload = userIds.map((ui) => {
     return {
       source: `main_feed_broad:${userId}`,
-      target: `user:${ui}`,
+      target: `${targetFeed}:${ui}`
     };
   });
-  return await cs.followMany(payload);
+  await cs.followMany(payload);
 };
 
-const unFollowMainFeedBroad = async (userId, userIds) => {
-  if (!userIds || userIds.length == 0) {
+const unFollowMainFeedBroad = async (userId, userIds, user_anon = false) => {
+  if (!userIds || userIds.length === 0) {
     return;
   }
 
+  const targetFeed = user_anon ? 'user_anon' : 'user';
   userIds.map(async (ui) => {
     // sent job to queue to avoid rate limit
     unFollowFeedProcessQueue.add({
-      feedName: "main_feed_broad",
-      userId: userId,
-      targetFeed: "user",
-      unfollowUserId: ui,
+      feedName: 'main_feed_broad',
+      userId,
+      targetFeed,
+      unfollowUserId: ui
     });
   });
-  return;
 };
 
-
 module.exports = {
   followMainFeedBroad,
   unFollowMainFeedBroad
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The module exports have been updated to reflect the new function names. This is a necessary change given the renaming of the functions.

Overall, these changes seem to improve the flexibility of the code by allowing anonymous users to be followed or unfollowed. However, please ensure that the removal of the return statements does not affect other parts of your codebase.


<!-- This is an auto-generated comment by OSS CodeRabbit -->